ironmon_tracker/MGBA.lua

```diff
@@ -538,18 +538,22 @@ MGBA.OptionMap = {
 		getText = function() return Resources.MGBA.OptionDevBranchUpdates end,
 	},
 	[11] = {
+		optionKey = "Override Button Mode to LR",
+		getText = function() return Resources.MGBA.OptionOverrideButtonModeLR end,
+	},
```

Comment:
Does mGBA need this option, and by extension should we be overriding the button mode on mGBA, considering the new shortcut does nothing on mGBA?

Reply:
Technically no, but since its here, im going to leave it. If anything, i find this "override to change the default setting from HELP to something useful" to be useful.